### DISCUSSION TITLE
[Sniffer] Constant notching mode: "Keep Notching"

### DISCUSSION THREAD

**Discussion by michcior2** [collin80/SavvyCAN#845] [Ideas]
I spent lot of time, clicking the "Notch"  button for minutes, just to clean all before looking for particular bits. There are so many changes, that simple button is not enough. So, what about constant notching? 
![obraz](https://github.com/user-attachments/assets/3c63774b-22b6-4c08-bfc9-390012060c03)

[keepnothing.zip](https://github.com/user-attachments/files/16992154/keepnothing.zip)

**Comment by collin80** [comment]
Yeah, I could see that being useful. Just click keep notching and leave it going for a while then deselect it after a while and anything new after that would then show up but anything that happened prior would be notched out. It seems useful. I will add it to the list of things that should get implemented.
